src/bootlint.js

```diff
@@ -937,12 +959,7 @@ var LocationIndex = _location.LocationIndex;
             var elem = $(this);
             var urlAttr = (tagNameOf(this) === 'LINK') ? 'href' : 'src';
             var pathSegments = parseUrl(elem.attr(urlAttr)).pathname.split('/');
-            var matches = pathSegments.map(function (segment) {
-                var match = segment.match(/^\d+\.\d+\.\d+$/);
-                return match ? match[0] : null;
-            }).filter(function (match) {
-                return match !== null;
-            });
+            var matches = getSemverFromURI(pathSegments);
             if (!matches.length) {
                 return;
             }
```

Comment:
This does give me an error when I try to return matches after the

```javascript
 if (!matches.length) {
  return;
}
```

gives me an error.
```javascript
211:12  error  Expected no return value                        consistent-return
```

I'm sure it's just dumb stuff I'm missing. Sorry for the issues with all this.

function:

```javascript
function getSemverFromURL(pathSegments) {
        var matches = pathSegments.map(function (segment) {
            var match = segment.match(/^\d+\.\d+\.\d+$/);
            return match ? match[0] : null;
        }).filter(function (match) {
            return match !== null;
        });

        if (!matches.length) {
            return;
        }
        else {
            return matches;
        }
    }
```